### DISCUSSION TITLE
GitHub Changes

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.docs#69] [General]
"Re: Pages Legacy Worker Sunset

Going forward, GitHub Actions will become a requirement for deploying Pages sites that require a build step, such as those utilizing Jekyll. We chose to implement this change because GitHub Actions provides developers a modern, secure sandbox that surpasses the capabilities of pagesworker. With GitHub Actions, you get your choice of operating system, log streaming, dependency caching, custom gates, and  modern CI/CD features."

Is the infrastructure for the Oqtane docs site affected by this GitHub change?

@iJungleboy

**Comment by iJungleboy** [comment]
The docs don't use an online build step. 

They are built using DocFx in the dev environment (I trigger it using VS Code, but Visual Studio should work as well). 

Github then just publishes the files 1:1.

But AFAIK the actions are already in use, just to push, so either way wouldn't seem to make any difference.
